apps/prairielearn/src/pages/instructorFileEditor/instructorFileEditor.sql

```diff
@@ -1,20 +1,21 @@
 -- BLOCK select_file_edit
 SELECT
-  fe.id,
-  fe.orig_hash,
-  fe.file_id,
-  floor(
-    DATE_PART('epoch', CURRENT_TIMESTAMP - fe.created_at) / 3600
-  ) AS age,
-  fe.job_sequence_id
+  fe.*
 FROM
   file_edits AS fe
 WHERE
   fe.user_id = $user_id
   AND fe.course_id = $course_id
   AND fe.dir_name = $dir_name
   AND fe.file_name = $file_name
-  AND fe.deleted_at IS NULL;
+  AND fe.created_at > (
+    CURRENT_TIMESTAMP - $max_age * interval '1 millisecond'
```

Comment:
Nit, using `make_interval` is slightly more conventional in our codebase. Note the `_secs` prefix to indicate a unit (we like to do this where possible). JS will have to be updated.

```suggestion
    NOW() - make_interval(secs => $max_age_sec)
```

Reply:
Done in c23337974 and 7d7fb12dc.